pyproject.toml

```diff
@@ -236,9 +236,9 @@ addopts = [
   "--ds=dandiapi.settings.testing",
 ]
 filterwarnings = [
-  # "error",
+  "error",
   # pytest often causes unclosed socket warnings
   "ignore:unclosed <socket\\.socket:ResourceWarning",
-  # https://github.com/celery/kombu/issues/1339
-  "ignore:SelectableGroups dict interface is deprecated:DeprecationWarning:kombu",
+  # TODO: Remove this with pytest_factoryboy
+  "ignore:Using a <class 'dict'> as model type:UserWarning:pytest_factoryboy.fixture",
```

Comment:
Is this warning not fixed via https://github.com/dandi/dandi-archive/pull/2571?

Reply:
No, unrelated. This is purely caused by `pytest_factoryboy` and use of `factory.Dict`.